libraries/src/Extension/ExtensionHelper.php

```diff
@@ -323,6 +322,7 @@ class ExtensionHelper
         ['plugin', 'sessiongc', 'task', 0],
         ['plugin', 'sitestatus', 'task', 0],
         ['plugin', 'updatenotification', 'task', 0],
+        ['plugin', 'stats', 'task', 0],
```

Comment:
```suggestion
        ['plugin', 'sitestatus', 'task', 0],
        ['plugin', 'stats', 'task', 0],
        ['plugin', 'updatenotification', 'task', 0],
```
Please alpha order plugins of the same folder.